### PR TITLE
ACMS-1240: Update Starter Kit to have toggle module feature

### PR DESCRIPTION
**Pull request by chandan-singh7929**

### CODE REVIEW

**Comment by chandan-singh7929** [comment]
@vishalkhode1 - I have not included following module as dependencies of common

- Acquia Connector
- Acquia Search
- Search API Solr

**Review by fiasco — CHANGES_REQUESTED** [review]

File: acquia_cms_common.info.yml [hunk 1]
```diff
@@ -4,21 +4,50 @@ description: "Handles shared functionality for Acquia CMS."
...
   - drupal:content_moderation
+  - drupal:content_translation
+  - drupal:diff
   - drupal:entity_clone
+  - drupal:field_ui
   - drupal:focal_point
+  - drupal:google_analytics
+  - drupal:google_tag
+  - drupal:history
+  - drupal:honeypot
```
[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply acquia_cms_common.info.yml:24]: @fiasco - Thank you for input.
@vishalkhode1 - As discussed I am removing these module from ` .file` but we will keep this in `composer.json` and move these dependency in acquia_cms module/profile accordingly.

File: acquia_cms_common.info.yml [hunk 2]
```diff
@@ -4,21 +4,50 @@ description: "Handles shared functionality for Acquia CMS."
...
+  - drupal:moderation_dashboard
+  - drupal:moderation_sidebar
+  - drupal:password_policy_character_types
+  - drupal:password_policy_length
+  - drupal:password_policy_username
   - drupal:pathauto
+  - drupal:purge_processor_cron
+  - drupal:purge_processor_lateruntime
+  - drupal:purge_ui
+  - drupal:recaptcha
```
[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply acquia_cms_common.info.yml:40]: Same for this one.

**Comment by chandan-singh7929** [comment]
@vishalkhode1 - As discussed, remove following modules dependencies from info file:
- drupal:google_analytics
- drupal:google_tag
- drupal:history
- drupal:honeypot
- drupal:recaptcha

**Pull request merged**